README.md

```diff
@@ -149,15 +149,35 @@ Custom parser must be a singleton class that has a `.call` method that accepts t
 
 There are several strategies to get tag statistics
```

Comment:
Readme is kinda messy right now, probably, it would require to rewrite it, I think, it's better be done with Benchmark results addition.

Reply:
@Bogda, close this comment?